frontend/packages/dev-console/integration-tests/features/addFlow/add-page.feature

```diff
@@ -0,0 +1,105 @@
+@add-flow
+Feature: Add page on Developer Console
+              As a user, I should be able select a way to create an Application, component or service from one of the options provided on Add page
+
+    
+        Background:
+            Given user is at developer perspective
+              And user has created or selected namespace "aut-add-page"
+              And user has installed OpenShift Serverless Operator
+              And user has installed OpenShift Pipelines operator
+              And user is at Add page
+
+        
+        @regression
+        Scenario: Getting started resources on Developer perspective
+             Then user will see Getting started resources
+              And user will see Create Application using Samples
+              And user will see Build with guided documentation
+              And user will see Explore new developer features
+
+        
+        @regression
+        Scenario: Getting started resources on Administrator perspective
+            Given user is at Administrator perspective
+              And user is at Overview page
+             Then user will see Getting started resources
+              And user will Set up your clcuster
+              And user will see Build with guided documentation
+              And user will see Explore new admin features
```

Comment:
This is not an "add page" feature? Shouldn't this part of another integration test?

Reply:
Removed that scenario